packages/dx-react-scheduler/docs/guides/grouping.md

```diff
@@ -0,0 +1,34 @@
+# React Scheduler - Grouping
+
+The React Scheduler allows you to group appointments by their [resources](../reference/resources.md). The group names are displayed by [GroupingPanel](../reference/grouping-panel.md) plugin.
+
+To change the group an appointment belongs to users can edit a corresponding appointment through the [AppointmentForm](../reference/appointment-form.md) or by drag-dropping the appointment with the help of [DragDropProvider](../reference/drag-drop-provider.md) plugin.
```

Comment:
Seems like this paragraph is not required because it has been described in resources guide and in this guide below.